resources/metadata/checkmarx.yaml

```diff
@@ -202,6 +202,8 @@ spec:
       - STAGES
       - STEPS
       default: FAILURE
+      possibleValues:
+        - FAILURE
```

Comment:
@nevskrem are there other possible values here? I just found the default.

Reply:
@CCFenner suspect this is based on Jenkins job result states and would therefore be UNSTABLE, FAILURE and SUCCESS.